### PR TITLE
Adding TailwindCSS + tailwind-ppx example

### PR DESCRIPTION
**Pull request by dylanirlbeck**
The PR should hopefully be self-explanatory --- I added some information about using Tailwind CSS and ReasonReact together.

cc @peterpme

### CODE REVIEW

File: website/i18n/en.json [hunk 1]
```diff
@@ -162,7 +165,7 @@
         "title": "useReducer Hook"
       },
       "usestate-event-value": {
-        "title": "Using event values with useState"
+        "title": "Using Event Values With useState"
```
Comment (dylanirlbeck) [inline website/i18n/en.json:168]: This is an auto-generated file, so I didn't see the point of removing this diff (even though it's not relevant to the PR). Let me know if I should fix.

File: docs/tailwind-css.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+Let's see an example of a React component using Tailwind:
+
+```reason
+[@react.component]
+let make = () =>
+  <div className="h-screen flex justify-center items-center">
+    <div className="max-w-sm rounded overflow-hidden shadow-lg p-4">
+      <img className="w-full" src=logo alt="Sunset in the mountains" />
+      <div className="px-6 py-4">
+        <div className="font-bold text-xl mb-2"> {"RE-Tailwind" |> str} </div>
```
Comment (peterpme) [inline docs/tailwind-css.md:31]:
```suggestion
        <div className="font-bold text-xl mb-2"> {React.string("Tailwind")} </div>
```

File: docs/tailwind-css.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        <p className="text-gray-700 text-base">
+          {"A reason react starter with tailwind" |> str}
```
Comment (peterpme) [inline docs/tailwind-css.md:33]:
```suggestion
          {React.string("A reason react starter with tailwind")}
```

File: docs/tailwind-css.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
+---
+title: Tailwind CSS
```
Comment (peterpme) [inline docs/tailwind-css.md:2]:
```suggestion
title: Styling: Tailwind CSS
```

File: docs/tailwind-css.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+          {"A reason react starter with tailwind" |> str}
+        </p>
+      </div>
+   </div>
+  </div>;
+```
+
+which gives us the following UI:
+
+<img src="/reason-react/img/tailwind-example.png">
```
Comment (peterpme) [inline docs/tailwind-css.md:42]: im not sur ethis is working is it?

Comment (dylanirlbeck) [reply docs/tailwind-css.md:42]: @peterpme The image works locally, and it's included in the diff of this PR. What makes you think it's not working, if I may ask?

Comment (peterpme) [reply docs/tailwind-css.md:42]: oh nvm i was just looking at the diff!

**Pull request merged**